test/functional/test_framework/util.py

```diff
@@ -516,32 +517,34 @@ def gen_return_txouts():
     for i in range(512):
         script_pubkey = script_pubkey + "01"
     # concatenate 128 txouts of above script_pubkey which we'll insert before the txout for change
-    txouts = "81"
+    txouts = []
+    from .messages import CTxOut
     for k in range(128):
-        # add txout value
-        txouts = txouts + "0000000000000000"
-        # add length of script_pubkey
-        txouts = txouts + "fd0402"
-        # add script_pubkey
-        txouts = txouts + script_pubkey
+        txout = CTxOut()
```

Comment:
You could take out everything from this hot loop except the `txtouts.append` line to see if we gain back that 0.18 secs?

Reply:
Strangely enough this made it 0.2 seconds slower on my machine :/ So I think it's better as-is.